<launch>

  <include file="$(find pr2_machine)/$(env ROBOT).machine" />
  <include file="$(find pr2_navigation_slam)/slam_gmapping.xml" />
  <include file="$(find pr2_navigation_teleop)/teleop.xml" />
  <include file="$(find pr2_navigation_perception)/lasers_and_filters.xml" />
  <include file="$(find pr2_navigation_perception)/ground_plane.xml" />
  <include file="$(find pr2_navigation_slam)/move_base.xml" />

  <!-- Arm navigation -->
  <include file="$(find pr2_3dnav)/right_arm_navigation.launch" />


  <!-- For object detection -->
  <node pkg="point_cloud_converter" name="point_cloud_converter2" type="point_cloud_converter">
    <remap from="/points_in" to="/wide_stereo/points" />
    <remap from="/points2_out" to="/wide_stereo/points2" />
  </node>


  <param name="cmvision/color_file" type="string" value="$(find litterbox)/colors.txt"/>-->

  <include file="../cmvision/cmvision.launch"/>

</launch>

